db/migrate/20241020010245_add_tool_name_to_ai_tools.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddToolNameToAiTools < ActiveRecord::Migration[7.1]
+  def up
+    add_column :ai_tools, :tool_name, :string, null: false, default: "", if_not_exists: true
+
+    # Migrate existing name to tool_name
+    execute <<~SQL
+      UPDATE ai_tools
+      SET tool_name = regexp_replace(LOWER(name),'[^a-z0-9_]','', 'g');
+    SQL
```

Comment:
re: We probably want a distinct constraint on Name which means we need to append (1,2,3 etc) onto name during the migration if there are dupes.

I reckon we should call it off and have another PR if we actually need it. Here, I only added the migration to migrate existing name to the new tool_name column. Let me know if you're ok with it.